packages/core/src/App/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -28,7 +28,10 @@ const RouteWithSubRoutes = route => {
             }
             result = <Redirect to={to} />;
         } else if (is_valid_route && route.is_authenticated && !route.is_logged_in && !route.is_logging_in) {
-            redirectToLogin(route.is_logged_in, getLanguage());
+            const url_search_params = new URLSearchParams(this.props.location.search);
```

Comment:
```suggestion
            const url_search_params = new URLSearchParams(window.location.search);
```
we can use the location from the global `window` object.